ogb/graphproppred/mol_encoder.py

```diff
@@ -5,12 +5,14 @@
 full_bond_feature_dims = get_bond_feature_dims()
 
 class AtomEncoder(torch.nn.Module):
-
-    def __init__(self, emb_dim):
+    def __init__(self, emb_dim, num_atoms_feat=None):
```

Comment:
why don't you name this as `optional_full_atom_features_dims`?

Reply:
Thanks for the feedback! I updated `num_atoms_feat` to `optional_full_atom_features_dims`.